src/containers/ToDoList/ToDoList.jsx

```diff
@@ -7,11 +7,43 @@ import {
   StyledWrapperSelectButtonGroupSeparated,
   StyledButtonClearCompletedTask,
   StyledWrapperNewTask,
+  StyledFilterButton,
 } from "./ToDoList.styles.js";
 import { useState } from "react";
 import { v4 as uuidv4 } from "uuid";
 import AddIcon from "@mui/icons-material/Add";
 
+const FilterType = {
+  ALL: "all",
```

Comment:
This line is not needed anymore I think.

Other than that this PR seems good to me. Once you remove it, I can approve it.